Crypto101.bib

```diff
@@ -282,3 +282,42 @@ @misc{schneier:ipsec
   year = {1999},
   note = {\url{https://www.schneier.com/paper-ipsec.pdf}}
 }
+@misc{cryptoeprint:2004:199,
+  author = {Xiaoyun Wang and Dengguo Feng and Xuejia Lai and Hongbo Yu},
+  title = {Collisions for Hash Functions MD4, MD5, HAVAL-128 and RIPEMD},
+  howpublished = {Cryptology ePrint Archive, Report 2004/199},
+  year = {2004},
+  note = {\url{http://eprint.iacr.org/2004/199}}
+}
+@INPROCEEDINGS{article143,
+  author =    {Bert den Boer and
+               Antoon Bosselaers},
+  title =     {{Collisions for the compression function of MD5}},
+  year =      {1993},
+  pages =     {293-304},
+  booktitle = {Advances in Cryptology - EUROCRYPT 1993},
+  editor =    {Tor Helleseth},
+  volume =    {765},
+  number =    {},
+  series =    {Lecture Notes in Computer Science},
+  address =   {Lofthus,N},
+  publisher = {Springer-Verlag}
+}
+@misc{cryptoeprint:2005:067,
+    author = {Arjen Lenstra and Xiaoyun Wang and Benne de Weger},
+    title = {Colliding X.509 Certificates},
+    howpublished = {Cryptology ePrint Archive, Report 2005/067},
+    year = {2005},
+    note = {\url{http://eprint.iacr.org/2005/067}},
+}
+@inproceedings{eurocrypt-2009-23793,
+  title={Cryptanalysis on HMAC/NMAC-MD5 and MD5-MAC},
+  booktitle={Advances in Cryptology - EUROCRYPT 2009, 28th Annual International Conference on the Theory and Applications of Cryptographic Techniques},
+  series={Lecture Notes in Computer Science},
+  publisher={Springer},
+  volume={5479},
+  pages={121-133},
+  doi={10.1007/978-3-642-01001-9_7},
+  author={Xiaoyun Wang and Hongbo Yu and Wei Wang and Haina Zhang and Tao Zhan},
+  year=2009
```

Comment:
This one's also published as PDF on IACR; please add a URL like the other one:

http://www.iacr.org/archive/eurocrypt2009/54790122/54790122.pdf